Loop54.Test.AspNetMvc/Controllers/GetRelatedEntitiesController.cs

```diff
@@ -38,6 +39,11 @@ public ActionResult Index(string id, string type)
             request.ResultsOptions.Skip = 0;
             request.ResultsOptions.Take = 20;
 
+            // set relation kind
+            RelationKinds relKind;
+            if(Enum.TryParse(relationKind, true, out relKind))
```

Comment:
If this is taken from radio buttons or a dropdown then invalid values should not occur, so should not be silently ignored here.

Reply:
Added error handling